include/deployer.bash

```diff
@@ -191,12 +191,26 @@ doctor() {
     docker-check-version
     compose-generate-check-diff verbose
     generate_uaa_check_diff verbose
+    if [[ "$(uname)" == Darwin* ]]; then
+      check-cbd-location
+    fi
 }
 
 cbd-find-root() {
     CBD_ROOT=$PWD
 }
 
+check-cbd-location() {
+  declare home_path=$(cd -P -- "$(dirname -- ~/.)" && pwd -P)
+  declare cbd_path=$(cd -P -- "$(dirname -- .)" && pwd -P)
+  let home_size=${#home_path}
+  if [[ $home_path != ${cbd_path:0:$home_size} ]]; then
+      echo "[ERROR] Please relocate deployer under your home folder. On OSX other locations are not supported." | red
```

Comment:
instead of `echo` you can use the `error` function, which prints to stderr with red color.